drivers/cc110x/include/cc110x-defaultsettings.h

```diff
@@ -33,10 +33,8 @@ extern "C" {
 extern const char cc110x_default_pa_table[8];
 #endif
 
-#ifndef CC110X_DEFAULT_FREQ
-#define CC110X_DEFAULT_FREQ cc110x_default_base_freq
-extern const char cc110x_default_base_freq[3];
-#endif
+extern const char cc110x_base_freqs[3][3];
+extern const char cc110x_channel_widths[3][2];
```

Comment:
Where is this used?

Reply:
`const char cc110x_channel_widths[3][3];` is a left over I forgot to delete. (I was considering to use different channel width depending on the base band as e.g. the 900MHz LoRa channels have a spacing of 2.16 MHz instead of the 50kHz currently used. But I dropped that idea.)

`const char cc110x_base_freqs[3][3];` is used [here](https://github.com/maribu/RIOT/blob/bcaf49d037103ce758cba60945ff0cc9f6c5f5b9/drivers/cc110x/cc110x.c#L113-L120):
``` C
void cc110x_set_band(cc110x_t *dev, cc110x_band_t band)
{
#if ENABLE_DEBUG == 1


    DEBUG("cc110x_set_band(): Setting band to %sMHz\n", cc110x_band_name[band]);
#endif
    cc110x_writeburst_reg(dev, CC110X_FREQ2, cc110x_base_freqs[band], 3);
}
```